src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy

```diff
@@ -362,7 +362,7 @@ class PitestTask extends JavaExec {
         map['includedTestMethods'] = optionalCollectionAsString(includedTestMethods)
         map['sourceDirs'] = (getSourceDirs().files*.absolutePath)?.join(',')
         map['detectInlinedCode'] = optionalPropertyAsString(detectInlinedCode)
-        map['timestampedReports'] = optionalPropertyAsString(timestampedReports)
+        map['timestampedReports'] = optionalPropertyAsString(timestampedReports) ?: 'false'
```

Comment:
It's not the best place for that kind of logic. PitestTask just passes the values. The default values can be set in [`PitestPlugin`](https://github.com/szpak/gradle-pitest-plugin/blob/12857274f550d5a21d570621934bb86d5af16c0f/src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy#L114).

Reply:
Thanks for clarifying. Updated the PR.